### ISSUE TITLE
Error on ant clean all because of missing package

### ISSUE THREAD

**Issue by ghost** [coveo/coveo-sap-commerce-connector#1]
I get an error when compiling my project with the <extension name='searchprovidercoveosearchservices'/>

Because of coveo/searchprovidercoveosearchservices/testsrc/com/coveo/indexer/service/impl/CoveoObjectTypeSnIndexerValueProviderTest.java

The test is trying to import com.coveo.core.model.ApparelProductModel;  but the package com.coveo.core is not part of the extension.

**Comment by jsonDaCruzo** [comment]
Hi @fbeaulieu-canac, thank you for raising this issue. I'm the Product Manager at Coveo responsible for our SAP integrations (including this connector). Apologies for not spotting this sooner, I've subscribed to alerts on issues opened in this repo, so will be more responsive if any others are opened in future. 

Just wanted to let you know that we'll be investigating this issue in our upcoming sprint and we'll keep you posted once we have a resolution.

**Comment by thetruechar** [comment]
Same error encountered:
```
 [yjavac] Compiling 50 source files to ~/SAPDevelop/cx/c4h340_2211/workspace/hybris/bin/custom/searchprovidercoveosearchservices/classes
   [yjavac] ~/SAPDevelop/cx/c4h340_2211/workspace/hybris/bin/custom/searchprovidercoveosearchservices/testsrc/com/coveo/indexer/service/impl/CoveoObjectTypeSnIndexerValueProviderTest.java:4: error: package com.coveo.core.model does not exist
   [yjavac] import com.coveo.core.model.ApparelProductModel;
   [yjavac]                            ^
   [yjavac] 1 error
```

**Comment by thetruechar** [comment]
@fbeaulieu-canac you can just delete the dir testsrc after the copy.

**Comment by jsonDaCruzo** [comment]
Hi @fbeaulieu-canac, @thetruechar, quick update to let you know that this has now been resolved in the latest version of the connector 3.1.4: https://github.com/coveo/coveo-sap-commerce-connector/tree/v3.1.4
